plugins/outputs/influxdb/sample.conf

```diff
@@ -4,6 +4,8 @@
   ##
   ## Multiple URLs can be specified for a single cluster, only ONE of the
   ## urls will be written to each interval.
+  ## To send every metrics into multiple influxdb,
+  ## define additional [[outputs.influxdb]] section.
```

Comment:
Can you please add this to the README instead of the sample.conf!? That's just true for many of the output plugins, so I wouldn't clutter the config with it. What do you think?

Reply:
Sounds good. The question is which README file would be the best as every plugin has own readme file :)

Reply:
Well, I would start with the readme for `outputs.influxdb` as that is the one you specifically wanted to address.

Not all outputs need this additional sections, so it doesn't apply to all.